firefox-ios/Client/Frontend/Browser/Tabs/Legacy/LegacyTabCell.swift

```diff
@@ -110,6 +110,8 @@ class LegacyTabCell: UICollectionViewCell,
         title.contentView.addSubview(self.favicon)
 
         setupConstraint()
+        
+        backgroundColor = .red
```

Comment:
@yusuphjoluwasen thanks for updating the font styles! Those changes look great, curious on setting this background color, do we need this change here?

Reply:
@cyndichin Thanks for pointing that out, I've made the necessary revisions.